kratos/python/add_containers_to_python.cpp

```diff
@@ -26,6 +26,8 @@
 #include "includes/convection_diffusion_settings.h"
 #include "includes/radiation_settings.h"
 #include "utilities/quaternion.h"
+// #include "includes/element.h" // NOTE: For NEIGHBOUR_ELEMENTS
+// #include "includes/condition.h" // NOTE: For NEIGHBOUR_CONDITIONS
```

Comment:
Why adding them if commented?

Reply:
Because I tried to expose it and it failed as a Isaid before due to a pybind error that I am unable to fix